src/core/server/http/index.ts

```diff
@@ -94,3 +94,4 @@ export {
 } from './cookie_session_storage';
 export * from './types';
 export { BasePath, IBasePath } from './base_path_service';
+export { validateObject } from './prototype_pollution';
```

Comment:
I would really like to avoid static functions leaking outside of core. If this needs to be used in plugins, maybe moving it to a package (existing or new) would make sense. wdyt @legrego?

Reply:
I'm not opposed to moving this into a package. That'd certainly make it easier to consume

Reply:
@DziyanaDzeraviankina we are currently discussing the details with @legrego. Note that if it's mandatory to have this merged for `7.11`, we could do the moving/refactoring at a later time. So considerer this approved on my side if this is urgent.

Reply:
I'm not sure if there's a suitable package for that, maybe it could be moved to `kbn-config-schema` package, or is it better to create a new one?

Reply:
Would `@kbn/std` work for this?